packages/integration/api-report.md

```diff
@@ -10,6 +10,8 @@ import { Config } from '@backstage/config';
 export class AzureIntegration implements ScmIntegration {
     constructor(integrationConfig: AzureIntegrationConfig);
     // (undocumented)
+    get annotationPrefix(): string;
+    // (undocumented)
```

Comment:
For all these `// (undocumented)`, I'd recommend either adding TODO so our IDEs or even the `@backstage/plugin-todo` picks them up, OR just adding them to the documentation via this PR :)  But the easiest may just be doing:

```
// TODO: Azure slug is not yet documented https://backstage.io/docs/features/software-catalog/well-known-annotations
```

Just may help elevate the fact that there is tech debt there, and I'm sure one of us can pick that up!

Reply:
what is the correct way to add this documentation? this file was auto-generated from the build. Is this created from jsdoc annotations in the code?

Reply:
Yep.